gender_novels/deployment/static/css/base.css

```diff
@@ -33,9 +39,24 @@ p {
 
 .content {
     background-color: #FCF5E9;
+    text-align: left;
+}
+
+.portrait {
```

Comment:
This unfortunately distorts the images. 
Optimally, we would just crop the images to be 500x500 pixels, that would assure a uniform look.
For the moment, you could try:
```
.portrait{
    max-width: 250px;
    max-height: 250px;
    height: auto;
    width: auto;
```